tests/lib/rook/base.py

```diff
@@ -92,19 +97,24 @@ def install(self):
             matcher=common.regex_count_matcher(pattern, 1),
             attempts=30, interval=10)
 
+        logger.info("Rook successfully installed and ready!")
+
+    def install_ceph(self, cluster_yaml='cluster.yaml', health_test=True):
         self.kubernetes.kubectl_apply(
-            os.path.join(self.ceph_dir, 'cluster.yaml'))
+            os.path.join(self.ceph_dir, cluster_yaml))
```

Comment:
I think CI is failing because you're prepending `ceph_dir` to `cluster_yaml`, when in `set_storage_pattern` you return the path including ceph_dir already.

Reply:
Good catch